examples/playground/json-examples/carto-bq-tiler.json

```diff
@@ -11,7 +11,7 @@
     {
       "@@type": "MapView",
       "controller": true,
-      "mapStyle": "https://basemaps.cartocdn.com/gl/positron-gl-style/style.json"
+      "mapStyle": "@@#CARTO_BASEMAP.POSITRON"    
```

Comment:
I feel that this might be too confusing as an example. There is no export called `CARTO_BASEMAP` from any module. It used to be more self-explanatory when a URL was explicitly used.

Reply:
You're right, rolling back. Thanks for the feedback

Reply:
After a second thought, I'm not 100% sure. There's an export called BASEMAP from `@deck.gl/carto`. If you for example want to change to `voyager` will be more direct because in the doc there is [a section](https://github.com/visgl/deck.gl/blob/master/docs/api-reference/carto/basemap.md#supported-basemaps) to explain it, whereas we have no doc to explain how the URLs are composed

Reply:
`BASEMAP` and `CARTO_BASEMAP` are two different keywords. If you look at the other constants used in the playground, `COORDINATE_SYSTEM` is a legit export from `@deck.gl/core`, and the user won't be able to find any documentation if it's renamed to `CORE_COORDINATE_SYSTEM`.

Reply:
What about if we rename it to `BASEMAP`?

Reply:
Overall, the playground can benefit from a "help" screen that explains how it works and how to port the code to an application.

Reply:
Can we use a new PR to discuss this change? It has nothing to do with update triggers.